File: flutter_application_1/test_api.dart
import 'dart:convert';
import 'package:http/http.dart' as http;

void main() async {
  final baseParams = 'appTypeId=5&langId=29&timezoneName=Europe%2FParis&userCountryId=135';
  final url = 'https://webws.365scores.com/web/games/allscores/?$baseParams&sportId=1';
  final response = await http.get(Uri.parse(url), headers: {
    'User-Agent': 'Mozilla/5.0',
    'Accept-Encoding': 'gzip, deflate, br'
  });
  
  final decoded = jsonDecode(response.body);
  if (decoded['games'] != null) {
    final games = decoded['games'] as List;
    for (final g in games) {
      if (g['statusGroup'] == 3) {
        print('Found Live Game:');
        print(g.keys.toList());
        print('gameTime: ${g['gameTime']}');
        print('gameTimeDisplay: ${g['gameTimeDisplay']}');
        print('gameTimeAndStatusDisplay: ${g['gameTimeAndStatusDisplay']}');
        print('actualActionTime: ${g['actualActionTime']}');
        print('startTime: ${g['startTime']}');
        print('playClock: ${g['playClock']}');
        print('clock: ${g['clock']}');
        print('lastActionTime: ${g['lastActionTime']}');
        break;
      }
    }
  }
}

File: flutter_application_1/test_api.py
import urllib.request
import json

url = 'https://webws.365scores.com/web/games/results/?appTypeId=5&langId=1&timezoneName=Europe/London&competitions=672&limit=50'
req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
try:
    with urllib.request.urlopen(req) as response:
        data = json.loads(response.read().decode())
        for g in data.get('games', []):
            print(f"Match ID: {g['id']} - {g['homeCompetitor']['name']} vs {g['awayCompetitor']['name']} {g['homeCompetitor']['score']}-{g['awayCompetitor']['score']}")
except Exception as e:
    print(e)
